src/main/java/hudson/plugins/matrix_configuration_parameter/MatrixCombinationsParameterValue.java

```diff
@@ -111,6 +114,7 @@ public int hashCode() {
         return result;
     }
 
+    @SuppressFBWarnings(value="EQ_GETCLASS_AND_CLASS_CONSTANT", justification="Should match only with MatrixCombinationsParameterValue")
```

Comment:
<BugInstance type='EQ_GETCLASS_AND_CLASS_CONSTANT' priority='Normal' category='BAD_PRACTICE' message='hudson.plugins.matrix_configuration_parameter.MatrixCombinationsParameterValue.equals(Object) fails for subtypes' lineNumber='122'/>